### DISCUSSION TITLE
Where does it store the screenshots?

### DISCUSSION THREAD

**Discussion by ErnestoBorio** [dirkwhoffmann/virtualc64#721] [Q&A]
Where does it store the screenshots?

**Comment by namwollem** [comment]
If anyone finds this out, it would be very useful? Surely someone knows?!

**Comment by dirkwhoffmann** [comment]
Just open the Screenshot browser (Machine menu) and hit this button:

<img width="918" alt="Bildschirmfoto 2024-02-20 um 21 45 28" src="https://github.com/dirkwhoffmann/virtualc64/assets/12561945/5ce78511-f820-44a5-a2f3-90c4df4b9403">

It will open `~/Library/Application Support/VirtualC64/Screenshots` which is where the screenshots are saved.
